src/widget/form/groupchatform.h

```diff
@@ -38,6 +41,9 @@ class GroupChatForm : public GenericChatForm
 
     void onUserListChanged();
     void peerAudioPlaying(int peer);
+    void resetLayout();
+
+    virtual void show(Ui::MainWindow &ui);
```

Comment:
@agilob Here's why it crashes, but I don't get yet why… When looking into implementation you'll find this is complete crap and the first thing we need to clean up.